Selenium.Axe/Selenium.Axe/AxeBuilder.cs

```diff
@@ -49,7 +67,10 @@ public AxeBuilder WithTags(params string[] tags)
         {
             ValidateParameters(tags, nameof(tags));
 
+#pragma warning disable CS0618
             Options = null;
```

Comment:
Would it be easier for a user to debug if this (and the similar cases in the other new builder methods) instead threw an exception if Options were non-null? (normally this would be a breaking change, but there hasn't been a release that includes the new option builder methods yet)

Reply:
this shouldnt be used with Options & that is already documented. We will also be removing this in upcoming releases, since it is obsolete.

Reply:
I agree that it shouldn't be used with Options; I'm saying that if someone missed that documentation, I think they would probably notice/understand/fix their issue faster if they got an exception explaining that they can't be used together, rather than having it "succeed" while silently ignoring some of their settings.